test/infrastructure/inmemory/templates/cluster-template-inmemory-development.yaml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
As we are publishing this file on our release. let's have a follow-up to add an actual working cluster-template here

Reply:
I wonder if we should do some sort of alpha release to verify this all works

Reply:
I have added the template I'm using for experimenting with the new provider. 
I will open a follow-up issue to improve it

Reply:
I think @ykakarap can update it when he's working on the ClusterClass support (with a template using a CC)